config/services.yaml

```diff
@@ -5,6 +5,7 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     package_name_pattern: '[A-Za-z0-9_.-]+/[A-Za-z0-9_./-]+?'
+    package_name_pattern_v2: '[A-Za-z0-9_.-]+/[A-Za-z0-9_./~-]+?'
```

Comment:
why `~`?

Reply:
You need to be able to pass through dev packages, and they contain a `~`

https://packagist.org/p2/symfony/polyfill-php72~dev.json

Reply:
I guess you could merge it in the previous pattern, but i wanted to make it obvious

Reply:
yes, we can combine it